product-scenarios/scenarios-common/src/main/java/org/wso2/am/scenario/test/common/APIPublisherRestClient.java

```diff
@@ -977,7 +978,6 @@ public void developSampleAPI(JSONObject swaggerJson,
                 HttpResponse apiPublishResponse = changeAPILifeCycleStatus(updateLifeCycle);
                 scenarioTestBase.verifyResponse(apiPublishResponse);
             }
-            Thread.sleep(15000);
```

Comment:
Shall we keep this here till we implement a proper solution

Reply:
+1. Fixed with ec0bc7d80f8664d001910d8a439e58506a7059e1